ruby/scripts/create-extractor-pack.sh

```diff
@@ -13,14 +13,14 @@ else
   exit 1
 fi
 
-"$CARGO" build --release
+(cd extractor && "$CARGO" build --release)
+extractor/target/release/generator --dbscheme ql/lib/ruby.dbscheme --library ql/lib/codeql/ruby/ast/internal/TreeSitter.qll
```

Comment:
What is the reason for replacing `cargo run` with an explicit path? This used to run `cross run ...` in case `$CARGO=cross`, not sure if it matters, but I wondered why it was changed.

Reply:
Ok, makes sense. I don't really mind, but I was curious ;-)